### PR TITLE
Use correct version of wave in package.json

### PR DESCRIPTION
**Pull request by nlopin**
**What:**
Fixes the version of the library in package.json
​
**Why:**
I don't know why but the version in package.json is out of sync. It probably happened when we moved it from the internal repo.

**Checklist:**

- [x] Ready to be merged

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@freenow/wave",
-    "version": "1.25.1",
+    "version": "1.1.0",
```
Comment (snapsnapturtle) [inline package.json:3]: Didn't we just release version 1.10.0?

**Review by snapsnapturtle — APPROVED** [review]

**Comment by duranmla** [comment]
I tend to feel this should be done like this. When going to https://semantic-release.gitbook.io/semantic-release/support/faq#why-is-the-package-jsons-version-not-updated-in-my-repository you can see that 

>  _@semantic-release/git plugin can be used to push the updated package.json as well as other files to the Git repository_

Maybe that's the actual change we want to do.

**Review by agonzalezUX — APPROVED** [review]

**Comment by duranmla** [comment]
I think we should close this pull request and wait for a more thoughtful implementation using semantic release.
Has this hasn't been merge yet I guess we are agree. @nlopin

**Comment by nlopin** [comment]
I've created a separate issue #119 to address this separately

**Pull request closed without merging**